<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>title</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addInhabitantButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить 
 жителя
 в комнату</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_client.png</normaloff>../images for project/icon_client.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculationButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рассчитать
сумму</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_dollar.png</normaloff>../images for project/icon_dollar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Busy_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Сделать
 занятым/
 незанятым</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_ch_mark.png</normaloff>../images for project/icon_ch_mark.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
